src/Resources/config/app/config.yml

```diff
@@ -0,0 +1,5 @@
+sylius_resource:
+    resources:
+        sylius_invoicing.invoice:
+            classes:
+                model: Sylius\InvoicingPlugin\Entity\Invoice
```

Comment:
`Invoice` is already defined as a resource in configuration tree: https://github.com/Sylius/InvoicingPlugin/blob/master/src/DependencyInjection/Configuration.php#L53